b2b-app/imports/api/courses/server/publications.js

```diff
@@ -30,3 +30,11 @@ Meteor.publish('id.courses', function (id) {
 Meteor.publish('all.courses', () => {
   return Courses.find({})
 })
+Meteor.publish('id.courses', (id) => {
```

Comment:
I had already added this publish function but it's fine, we will remove it later.